docs/shared/ww3.yaml

```diff
@@ -0,0 +1,12 @@
+ww3:
+  namelist:
+    base_file: "{{ user.ww3_app }}/ww3/ww3_shel.inp.IN"
```

Comment:
Is the reference to (and exitence) of the `user.ww3_app` useful here given that it's only used once? IMO this is useful when it DRYs out the config, but is maybe distracting when there's no duplication anyway.

Reply:
Hmm, thats fair - I liked the distinct installation path but it is unnecessary here